.github/workflows/pull-build-image.yaml

```diff
@@ -3,6 +3,20 @@ name: Pull Build Image
 on:
   pull_request_target:
     types: [opened, edited, synchronize, reopened, ready_for_review]
```

Comment:
Can we delete edited event type?
Edit: triggers if description, title any other metadata is edited.